### DISCUSSION TITLE
Trouble in loading the .pbuf models.

### DISCUSSION THREAD

**Discussion by Chandan0404** [securefederatedai/openfederatedlearning#345] [Q&A]
Hi all,
 
We are trying to implement the openFL on a tensorflow version for 3D tumor segmentation.
We did a successful sample run with 4 collaborators and completed the training.
However, the trained models are being saved in a “.pbuf ” format.
We are facing difficulties in loading these trained models and using it for testing.
 
Can you share some valuable suggestions to help us in this regard ?

**Comment by alexey-gruzdev** [comment]
Hey @Chandan0404 ! Thanks for the interest in OpenFL.

I believe you can utilize `Director-based workflow` and especially the method `best_model = fl_experiment.get_best_model()` like it was done in [this notebook](https://github.com/intel/openfl/blob/develop/openfl-tutorials/interactive_api/PyTorch_Kvasir_UNet/workspace/PyTorch_Kvasir_UNet.ipynb)
